persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -550,7 +551,7 @@ instance PGFF.FromField Unknown where
         Just dat -> return (Unknown dat)
 
 instance PGTF.ToField Unknown where
-    toField (Unknown a) = PGTF.Escape a
+    toField (Unknown a) = PGTF.Plain $ BB.byteString a
```

Comment:
@ldub Can you link to the corresponding lines in `persist-mysql` and `persist-sqlite` for comparison?

Reply:
e.g. whatever handles serializing `PersistDbSpecific` in those packages.

Reply:
Yeah, they are linked at the top of my PR description, but here they are again:

Currently, the implementation of `PersistDbSpecific`:
- (in persistent-mysql) ***does not*** escape the given ByteString: https://github.com/yesodweb/persistent/blob/master/persistent-mysql/Database/Persist/MySQL.hs#L239
- (in persistent-sqlite) ***does not*** escape the given ByteString: https://github.com/yesodweb/persistent/blob/master/persistent-sqlite/Database/Sqlite.hs#L470
- (in persistent-postgresql) ***does*** escape the given ByteString (with single quotes): https://github.com/yesodweb/persistent/blob/master/persistent-postgresql/Database/Persist/Postgresql.hs#L553

Reply:
Or did you mean linking to them directly in the code as a comment?

Reply:
No, I just jumped straight into the code changes w/o reading the PR description. Sorry for making extra work for you 😓